tutorials/nservicebus-sagas/2-timeouts/tutorial.md

```diff
@@ -10,10 +10,12 @@ Being able to model the concept of time as part of a long-running process is inc
 
 > For more on the difficulties associated with batch jobs, see [Death to the Batch Job](https://particular.net/blog/death-to-the-batch-job).
 
-NServiceBus gives us the ability to send messages, but NServiceBus sagas give us the ability to *send messages into the future*.
+NServiceBus gives us the ability to not only send messages, but also [delay messages](/nservicebus/messaging/delayed-delivery). Delayed delivery gives us the ability to *send messages into the future* which is used by [Saga Timeouts](/nservicebus/sagas/timeouts).
 
 No need to write batch jobs to query every record in the database every night. Instead, each instance is able to manage time in its own workflow, setting virtual alarm clocks to be awakened whenever needed.
 
+NOTE: Timeouts are guaranteed to be delivered after the passed point in time but might be delayed if the system is very busy.
```

Comment:
I'm not sure this reads very well, but I don't have a better proposal right now. I usually explain timeouts as: it's a precise scheduling, it's a not before the expected timeout.